mat/cdense.go

```diff
@@ -65,6 +65,27 @@ func NewCDense(r, c int, data []complex128) *CDense {
 	}
 }
 
+// ReuseAs changes the receiver if it IsEmpty() to be of size r×c.
+//
+// ReuseAs re-uses the backing data slice if it has sufficient capacity,
+// otherwise a new slice is allocated. The data is then zeroed.
```

Comment:
I would write, "The backing data is zero on return." rather than "is then zeroed", since this current wording suggests that it's zeroed after allocation as well, which is should not be as the runtime has already done that.

Reply:
Done.

Reply:
Also fixed the other ReuseAs methods